### PR TITLE
Feature project based php

### PR DESCRIPTION
**Pull request by dittertp**

### CODE REVIEW

File: roles/php/tasks/macos/install-php.yml [hunk 1]
```diff
@@ -56,4 +56,17 @@
     src: "macos/valet-sh.conf.j2"
     dest: "{{ valet_sh_prefix }}/etc/{{ current_php }}/conf.d/95-valet-sh.ini"
     mode: '0644'
-  notify: "brew | {{ current_php }} | restarted"
\ No newline at end of file
+  notify: "brew | {{ current_php }} | restarted"
+
+- name: "create symlink for default php"
+  file:
+    src: "/usr/local/bin/php7.3"
```
Comment (sippsolutions) [inline roles/php/tasks/macos/install-php.yml:68]: will this always be php7.3?

Comment (dittertp) [reply roles/php/tasks/macos/install-php.yml:68]: Only set for the "install" process. The php version will be changed automatically after the "install" command to your local default version

Comment (dittertp) [reply roles/php/tasks/macos/install-php.yml:68]: have thought about it. the symlink is now only created if it does not exist

**Comment by dittertp** [comment]
duplicate #112

**Pull request closed without merging**